homework/05LessonFive.js

```diff
@@ -63,7 +68,7 @@ for (forLoopCounter = 0; forLoopCounter > -10; forLoopCounter--) {
 
 // Problem 8:
 // Fill in forLoopArrayResult so that it matches forLoopArray
-var forLoopArrayResult = [];
+var forLoopArrayResult = [0,-1,-2,-3,-4,-5,-6,-7,-8,-9];
 
 // ****
```

Comment:
great work answering all the questions! There is a concept question that needs some answering down below--->

 // Write your answer in comments
 //
 // What are loops? Why do we use them?
 //